presidio-image-redactor/Pipfile

```diff
@@ -12,6 +12,7 @@ pydicom = ">=2.3.0,<3.0.0"
 pypng = ">=0.20220715.0,<1.0.0"
 python-gdcm = ">=3.0.22,<4.0.0"
 matplotlib = ">=3.6.2,<4.0.0"
+opencv-python = ">=4.8.0"
```

Comment:
With the additional of OpenCV, please check the license and add it to https://github.com/microsoft/presidio/blob/main/NOTICE if available for commercial use.

Reply:
Also, I believe you should update the pip lock file as well.

Reply:
Agree, NOTICE should be updated. We removed pipfile.lock from the different packages due to the different issues it caused.